Dockerfile

```diff
@@ -13,96 +8,64 @@ COPY modules/ modules/
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
-    git build-essential libxml2-dev libxslt-dev libpq-dev
-COPY --chown=vets-api:vets-api config/freshclam.conf docker-entrypoint.sh ./
-USER vets-api
-# XXX: this is tacky
-RUN freshclam --config-file freshclam.conf
-RUN gem install vtk
-ENTRYPOINT ["/usr/bin/dumb-init", "--", "./docker-entrypoint.sh"]
+
+
+# Install fwdproxy.crt into trust store
+# Relies on update-ca-certificates being run in following step
+COPY config/ca-trust/*.crt /usr/local/share/ca-certificates/
+
+# Download VA Certs
+COPY ./import-va-certs.sh .
+RUN ./import-va-certs.sh
+
+COPY config/clamd.conf /etc/clamav/clamd.conf
+
+RUN mkdir -p /clamav_tmp && \
+    chown -R nonroot:nonroot /clamav_tmp && \
+    chmod 777 /clamav_tmp
+
+
+ENV LANG=C.UTF-8 \
+   BUNDLE_JOBS=4 \
+   BUNDLE_PATH=/usr/local/bundle/cache \
+   BUNDLE_RETRY=3
+
 RUN gem install bundler:${BUNDLER_VERSION} --no-document
 
-###
-# build stage; use --target=builder to stop here
-# Also be sure to add build-args from development stage above
-#
-# This is development with the app copied in and built.  The build results are used in
-# prod below, but also useful if you want to have a container with the app and not
-# mount your local checkout.
-###
-FROM development AS builder
-# XXX: move modules/ to seperate repos so we can only copy Gemfile* and install a slim layer
-ARG bundler_opts
-
-COPY --chown=vets-api:vets-api Gemfile Gemfile.lock ./
-COPY --chown=vets-api:vets-api --from=modules /tmp/modules modules/
-
-RUN bundle install --binstubs="${BUNDLE_APP_CONFIG}/bin" $bundler_opts \
+COPY --from=modules /tmp/modules modules/
+COPY Gemfile Gemfile.lock ./
+RUN bundle install \
   && rm -rf /usr/local/bundle/cache/*.gem \
   && find /usr/local/bundle/gems/ -name "*.c" -delete \
   && find /usr/local/bundle/gems/ -name "*.o" -delete \
   && find /usr/local/bundle/gems/ -name ".git" -type d -prune -execdir rm -rf {} +
+COPY --chown=nonroot:nonroot . .
+
+EXPOSE 3000
+
+USER nonroot
 
-COPY --chown=vets-api:vets-api . .
-USER vets-api
-
-###
-# prod stage; default if no target given
-# to build prod you probably want options like below to get a good build
-# --build-arg sidekiq_license="$BUNDLE_ENTERPRISE__CONTRIBSYS__COM" --build-arg rails_env=production --build-arg bundler_opts="--no-cache --without development test"
-# This inherits from base again to avoid bringing in extra built time binary packages
-###
-FROM base AS production
-
-ENV RAILS_ENV=production
-COPY --from=builder $BUNDLE_APP_CONFIG $BUNDLE_APP_CONFIG
-COPY --from=builder --chown=vets-api:vets-api /srv/vets-api/src ./
-COPY --from=builder --chown=vets-api:vets-api /srv/vets-api/clamav/database ../clamav/database
-RUN if [ -d certs-tmp ] ; then cd certs-tmp ; for i in * ; do cp $i /usr/local/share/ca-certificates/${i/pem/crt} ; done ; fi && update-ca-certificates
-USER vets-api
-ENTRYPOINT ["/usr/bin/dumb-init", "--", "./docker-entrypoint.sh"]
```

Comment:
Can we remove the docker-entrypoint.sh file now?

Reply:
I will remove it in the k8s-> master PR (along with some other clean up tasks)